Project/Packages/manifest.json

```diff
@@ -36,6 +36,6 @@
     "com.unity.modules.vr": "1.0.0",
     "com.unity.modules.wind": "1.0.0",
     "com.unity.modules.xr": "1.0.0",
-    "com.unity.barracuda": "0.3.2-preview"
```

Comment:
Still needs barracuda too, right?

Reply:
Not since we depend on it in our package.